index.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
 var clearDirectory = require('./clear').clearDirectory;
+var log = require('fancy-log');
```

Comment:
Can you please name it the same way in both files?

Reply:
hi, sorry for the really delayed answer. 
I renamed it now. 

I wondered about this difference as well during implementing it, but it wasn't named the same before, so I continued naming it differently.